prdoc/pr_2657.prdoc

```diff
@@ -0,0 +1,28 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "[frame] `#[pallet::composite_enum]` improved variant count handling + removed `pallet_balances`'s `MaxHolds` config"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      The implementation of `VariantCount` trait for the result composite enums
+      like `RuntimeHoldReason` and `RuntimeFreezeReason` was fixed
+      and is now calculated as the sum of the `VariantCount::VARIANT_COUNT` of all corresponding `#[pallet::composite_enum]`s.
+      The Balances pallet's `Config` item `type MaxHolds` was removed
+      and `type Holds` are now bounded to the composite enum `RuntimeHoldReason` variant count,
+      so the runtime does not need to consider setting correct value for `MaxHolds`.
+
+      The implementation of the `VariantCount` trait for resulting composite enums,
```

Comment:
Seems duplicate?

Reply:
thank you, fixed, https://github.com/paritytech/polkadot-sdk/pull/2657/commits/4015ea09b70951dbdd2f09c1c1dc1cee6a162c98